src/github.com/vharitonsky/iniflags/.gitignore

```diff
@@ -0,0 +1 @@
+tags
```

Comment:
`iniflags` is only used by `http-proxy-lantern`. I guess it sneaked in by `go get`